man/adoc/bpftrace.adoc

```diff
@@ -3465,52 +3460,38 @@ See src/attached_probe.cpp:find_vmlinux() for details.
 
 === Debug Output
 
-The `-d` option produces debug output and does not run the program. This is mostly useful for debugging issues with bpftrace itself.
-You can also use `-dd` to produce a more verbose debug output, which will also print unoptimized IR.
+The `-d STAGE` option produces debug output. It prints the output of the
+bpftrace execution stage given by the _STAGE_ argument. The option can be used
+multiple times (with different stage names) and the special value `all` prints
+the output of all the supported stages.
+
+This option does a dry run, i.e. after the information is printed, bpftrace is
+terminated.
 
 Note: This is primarily used for bpftrace developers.
 
-The output begins with `Program` and then an abstract syntax tree (AST) representation of the program.
+The supported stages are:
```

Comment:
Nit: I might change the table header to 'The supported options are' so you can also include 'all'.

Reply:
Yeah, good idea, having everything in the table is more comprehensive.